{
  "name": "restful-mongo-http-handlers",
  "version": "2.0.1",
  "description": "## Test",
  "main": "index.js",
  "scripts": {
    "test": "DB_URL=mongodb://127.0.0.1:27017/test mocha -t 10000 ./test/**/spec*.js"
  },
  "repository": {
    "type": "git",
    "url": "git+ssh://git@github.com/wonderflow-bv/restful-mongo-http-handlers.git"
  },
  "author": {
    "name": "luca",
    "email": "luca@wonderflow.co"
  },
  "license": "MIT",
  "dependencies": {
    "bson-objectid": "^1.1.4",
    "logb": "0.0.2",
    "mongodb": "1.4.9"
  },
  "devDependencies": {
    "body-parser": "^1.15.0",
    "chai": "^3.5.0",
    "deferred": "^0.7.5",
    "express": "^4.13.4",
    "pow-mongodb-fixtures": "0.12.0",
    "supertest": "^1.2.0"
  },
  "readme": "# restful-mongo-http-handlers\n\n## Test\n\n1. `$ npm run test`\n2. The first time you run the test it will fail. That's because the id of the user in the api test does not exist yet.\n3. `$ mongo --port 27101`\n4. `> use local`\n5. `db.users.find()`\n6. Select the first user id and replace it in the test spec\n7. Run the test again\n",
  "readmeFilename": "README.md",
  "gitHead": "57bbc794322da95d3143dafb1fad7b6ae5cfdea7",
  "bugs": {
    "url": "https://github.com/wonderflow-bv/restful-mongo-http-handlers/issues"
  },
  "homepage": "https://github.com/wonderflow-bv/restful-mongo-http-handlers#readme",
  "_id": "restful-mongo-http-handlers@2.0.1",
  "_shasum": "9e47abd47367340125582028f5ac738a44a4ee66",
  "_from": "restful-mongo-http-handlers@latest"
}
